Find a functionally equivalent code:android.app.ContextImpl.startActivityAsUser:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,3 +1,6 @@
 {
-    startActivityAsUser(intent, null, user);
+    try {
+        ActivityManagerNative.getDefault().startActivityAsUser(mMainThread.getApplicationThread(), getBasePackageName(), intent, intent.resolveTypeIfNeeded(getContentResolver()), null, null, 0, Intent.FLAG_ACTIVITY_NEW_TASK, null, options, user.getIdentifier());
+    } catch (RemoteException re) {
+    }
 }

